src/autoscaler/go.mod

```diff
@@ -1,6 +1,6 @@
 module autoscaler
 
-go 1.15
+go 1.16
```

Comment:
This is likely to cause issues until https://github.com/cloudfoundry/app-autoscaler-ci/blob/126386f314db9fe93827f429cbfb3667ac96640a/dockerfiles/autoscaler-test/Dockerfile#L43-L51 is updated (assume the deploy image needs the same)